mogc_partnerships/views.py

```diff
@@ -72,11 +74,6 @@ def get_queryset(self):
         )
 
     def perform_create(self, serializer):
-        partner = serializer.validated_data["partner"]
-        if self.request.user not in partner.managers.all():
-            self.permission_denied(
-                self.request, f"Cannot create cohort for {partner.slug}"
-            )
```

Comment:
We'll lose this custom permissions error message with these changes, not sure if there is a way to maintain it using django-rest-framework permission classes.

Reply:
Okay, thanks for calling this out! I think we can live with it.